docs/Installation.md

```diff
@@ -16,7 +16,7 @@
 ### Launch the server
 
 First at all, you must open a terminal and launch the server
```

Comment:
First of all

Reply:
Fait une suggestion :)

Reply:
Juste "First of all" plutôt que "First at all" :p